src/constants.ts

```diff
@@ -4,14 +4,10 @@
  *--------------------------------------------------------------------------------------------*/
 'use strict';
 
-import * as nls from 'vscode-nls';
-
```

Comment:
nls.loadMessageBundle had issue but wasn't able to figure out immediately, As we are not supporting multi languages  at the moment, disabling this error causing localize